Gemfile.lock

```diff
@@ -338,7 +338,7 @@ GEM
     reverse_markdown (2.0.0)
       nokogiri
     rexml (3.2.4)
-    rubocop (1.10.0)
+    rubocop (1.11.0)
```

Comment:
Just as a reminder, you probably also want to update the Rubocop version here: https://github.com/exercism/website/blob/main/.github/workflows/tests.yml#L28